spec/dummy/app/views/welcome_mailer/welcome_email.html.erb

```diff
@@ -0,0 +1,2 @@
+<link rel="stylesheet" href="/assets/application.css"/>
```

Comment:
Can we move this to the layout?

Reply:
I don't think there's a layout for emails? The purpose of this mailer view is to test inlining (see `spec/integration/delivery_spec.rb`)

Reply:
Isn't this view rendered inside `spec/dummy/app/views/layouts/mailer.html.erb`?